docs/develop/testing.md

```diff
@@ -78,7 +81,11 @@ Alternatively, you can run each of the containers manually as per [dev setup](se
 
 To use FastAPI's WebGUI to call the endpoints easily, navigate to
 
-    http://localhost:8000/docs
+``` yaml
+https://[DOMAIN]/api/docs # (1)!
+```
+
+1.  if you are using the [dev setup](setup.md), you can access it at `http://localhost:8000/docs`
```

Comment:
is this supposed to be an annotation? it doesn't seem to be rendering correctly

Reply:
Fixed. Thanks!